index.html

```diff
@@ -961,13 +961,26 @@ <h2><dfn>Coalesced events</dfn></h2>
                 of the pointer movement.</figcaption>
             </figure>
 
-            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced event list</dfn> (a list of
+            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). If this event is a <code>pointermove</code>
             or <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.</p>
 
-            <p>The events in the coalesced event list will have increasing
-            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            <p>When the <code>getCoalescedEvents()</code> method is called, user agents MUST apply the following
+               steps to each event in the coalesced events list, before returning the list:</p>
+            <ul>
+                <li>set <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>,
+                    <a href="https://dom.spec.whatwg.org/#dom-event-istrusted"><code>isTrusted</code></a>,
+                    and <a href="https://dom.spec.whatwg.org/#event-target"><code>target</code></a> to match the
+                    dispatched "parent" pointer event that the <code>getCoalescedEvents()</code> method was called on</li>
+                <li>set <code>cancelable</code> and <code>bubbles</code> to <code>false</code>
+                    (as these events will never be dispatched in isolation)</li>
+                <li>set <a>coalesced events list</a> and <a>predicted events list</a> to an empty list</li>
```

Comment:
Does this mean the developer can't get the predicted events if they call getCoalescedEvents first? This seems surprising. Why do we need to set an empty list when the user calls getCoalescedEvents? It seems like they should be able to get the same coalesced event list multiple times if they choose to. Otherwise, you could have multiple event listeners conflict with each other if one inadvertently clears the coalesced event list.

Reply:
these are the coalesced and predicted lists properties of the coalesced/predicted events. otherwise, you get some weird recursion type stuff here... "the events that were coalesced to get the coalesced event"?

Reply:
would adding "set ... for the event to an empty list" help or seem redundant, as these are already steps scoped to "apply the following  steps to each event in the coalesced events list" ?

Reply:
Thanks for the explanation, I think this is fine as is and I just failed to read the context correctly.